packages/react/README.md

```diff
@@ -62,11 +71,18 @@ In addition to the feature provided by the [web sdk](https://openfeature.dev/doc
 npm install --save @openfeature/react-sdk
 ```
 
+#### yarn
```

Comment:
README change just to prompt a new release, but also we were missing the yarn section.